### PR TITLE
standardize all code samples

### PR DESCRIPTION
**Pull request by kbsali**
very small / detail PR to standardize the code samples throughout the doc.

### CODE REVIEW

File: core/data-providers.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ The `getCollection` method must return an `array`, a `Traversable` or a [`ApiPla
 If no data is available, you should return an empty array.
 
 ```php
+<?php
```
Comment (dunglas) [inline core/data-providers.md]: We don't do that in the Symfony doc but maybe it's clearer for newcomers. WDYT @api-platform/core-team

Comment (kbsali) [inline core/data-providers.md]: either way, but it should be the same everywhere (`<?php` was there in some examples, others not...).

Should i remove them all?

Comment (soyuka) [inline core/data-providers.md]: I think it's easier to copy/paste so doesn't seam like a bad idea to me

Comment (dunglas) [inline core/data-providers.md]: I agree, keep them.

**Comment by kbsali** [comment]
up!

**Comment by theofidry** [comment]
Thanks @kbsali

**Pull request merged**